rust/crd/src/lib.rs

```diff
@@ -897,7 +897,7 @@ mod tests {
           name: simple-trino
         spec:
           image:
-            productVersion: "455"
+            productVersion: "469"
```

Comment:
```suggestion
            productVersion: "470"
```

Reply:
You'd need to grep through the stuff and replace `469` with `470`

Reply:
Done in dbf9409e08c5b82aff7d4724af1fa5b014e96fbf